### PR TITLE
Support for cht-conf v3.19.0 and above

### PR DESCRIPTION
**Pull request by kennsippell**
#243

### CODE REVIEW

File: test/collateral/project-with-source/package.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
   "version": "1.0.0",
   "description": "",
   "devDependencies": {
-    "cht-conf": "^3.9.3"
+    "cht-conf": "github:medic/cht-conf#main"
```
Comment (kennsippell) [inline test/collateral/project-with-source/package.json:6]: pin this after publish

File: test/collateral/project-with-source/app_settings.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
...
+    "isDeclarative": true,
```
Comment (kennsippell) [inline test/collateral/project-with-source/app_settings.json:10]: this shouldn't be here

File: test/rules-engine.spec.js [hunk 3]
```diff
@@ -6,186 +6,204 @@ const Harness = require('../src/harness');
...
-        const scheduledDate = '2000-01-07';
-        const initialResult = await harness.fillForm('pnc_followup', ['no'], ['yes', scheduledDate]);
-        expect(initialResult.errors).to.be.empty;
+      before(async () => { return await harness.start(); });
+      after(async () => { return await harness.stop(); });
+      beforeEach(async () => { return await harness.clear(); });
+      afterEach(() => { expect(harness.consoleErrors).to.be.empty; });
 
-        await harness.setNow(scheduledDate);
+      it('support for cht-conf prior to v3.18.3', async () => {
```
Comment (kennsippell) [inline test/rules-engine.spec.js:203]:
```suggestion
      it('support for cht-conf prior to v3.19.0', async () => {   
```

File: test/collateral/project-with-source/app_settings.json (hunk 2 above)
Comment (kennsippell) [reply test/collateral/project-with-source/app_settings.json:10]: yes it should! no worries

**Review by eljhkrr — APPROVED** [review]
LGTM

**Pull request merged**